Components/Lists/list.css

```diff
@@ -0,0 +1,26 @@
+.element-display{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 1rem;
+    border-radius: 10px;
+    gap:15px;
```

Comment:
Any specific reason for using px here?

Reply:
No specific reason, will try to be consistent with the units